nixnet/_session/signals.py

```diff
@@ -17,6 +18,18 @@ def __repr__(self):
     def _create_item(self, handle, index, name):
         return Signal(handle, index, name)
 
+    @property
+    def resamp_rate(self):
```

Comment:
This should have also broken the tests
https://github.com/ni/nixnet-python/blob/master/tests/test_frames.py#L222